vmware_dvs/agent/firewalls/vcenter_firewall.py

```diff
@@ -92,24 +87,29 @@ def security_group_updated(self, action_type, sec_group_ids,
                                device_id=None):
         pass
 
-    def _get_dvs_for_port_id(self, port_id, p_key=None):
-        # Check if port is already known
+    def _update_dvs_port_map(self, ports):
```

Comment:
No, this functions calls from 2 places: in _apply_sg_rules_for_port function(line 75) and in get_dvs_for_port_id(line 62)